stripe/api_requestor.py

```diff
@@ -63,9 +63,11 @@ def _build_api_url(url, query):
 
 class APIRequestor(object):
 
-    def __init__(self, key=None, client=None, api_base=None, account=None):
+    def __init__(self, key=None, client=None, api_base=None, api_version=None,
+                 account=None):
         self.api_base = api_base or stripe.api_base
         self.api_key = key
+        self.api_version = api_version or stripe.api_version
```

Comment:
Here I basically made `api_version` injectable like all the other key attributes. I copied the pattern laid down by `api_base`.